### PR TITLE
Add hardness var to mineral walls

### PR DESCRIPTION
**Pull request by Mooshimi**
## About The Pull Request

- adds the hardness var to each mineral wall I could see that it was missing from, with the hardness based on Moh's scale
- lowered bamboo and wood hardness to 80.

## Why It's Good For The Game

- The var was created to have the probability of hulks breaking through walls on hit or not, this just makes it more used for the differences with walls

## Changelog

:cl:
fix: Fixed some strengths on mineral walls, making them harder to break through via hulk and mechs.
/:cl:

### CODE REVIEW

**Comment by Tennessee116** [comment]
>adds the **hardness var** to each mineral wall...

>refactor: Added the **density var** to mineral walls, which reflects the probability of hulks breaking them per hit.

I wouldn't point this out for any reason except that `density` is a reserved variable with an entirely different usage.

File: code/game/turfs/closed/wall/mineral_walls.dm [hunk 1]
```diff
@@ -55,6 +58,7 @@
 	icon_state = "bananium_wall-0"
 	base_icon_state = "bananium_wall"
 	sheet_type = /obj/item/stack/sheet/mineral/bananium
+	hardness = 95 //it's banana
```
Comment (JohnFulpWillard) [inline code/game/turfs/closed/wall/mineral_walls.dm:61]: it's still a mineral, should still be hard.

Comment (Mooshimi) [reply code/game/turfs/closed/wall/mineral_walls.dm:61]: how hard....?

Comment (JohnFulpWillard) [reply code/game/turfs/closed/wall/mineral_walls.dm:61]: That's up to you, not like it exists in real life, I just don't think it should be weaker than snow and bamboo...

Comment (Mooshimi) [reply code/game/turfs/closed/wall/mineral_walls.dm:61]: roger that captain

**Pull request merged**